packages/hooks/src/useLandingCompanyDetails.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@deriv/api';
+import { useMemo } from 'react';
+
+type TUseLandingCompanyDetailsPayload = Parameters<typeof useQuery<'landing_company_details'>>['1']['payload'];
+
+/** A custom hook that returns the available landing companies of the user's country. */
+const useLandingCompanyDetails = (payload: TUseLandingCompanyDetailsPayload) => {
+    const { data, ...rest } = useQuery('landing_company_details', {
+        payload,
+        options: { enabled: Boolean(payload.landing_company_details) },
+    });
```

Comment:
Should we use Api here? Since as Yashim mentioned that we should stop using hooks from `@deriv/api`

Reply:
I think Yashim mentioned to not use the @deriv/api because there is Logout issue when we use deriv/api/hooks with mobx stores.
@yashim-deriv @mahdiyeh-deriv

Reply:
He mentioned not to use hooks from `@deriv/api` however it's fine to use from `@deriv/hooks` as they rely on store under the hood

Reply:
yeah, this is fine. the `useQuery` has some fallback for legacy WS built in